Makefile

```diff
@@ -169,8 +169,11 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 GOIMPORTS ?= $(LOCALBIN)/goimports
 
 ## Tool Versions
+# renovate: datasource=repology depName=kustomize packageName=scoop/kustomize versioning=loose
 KUSTOMIZE_VERSION ?= v5.0.3
+# renovate: datasource=go depName=sigs.k8s.io/controller-tools
 CONTROLLER_TOOLS_VERSION ?= v0.12.0
+# renovate: datasource=go depName=golang.org/x/tools/cmd/goimports
```

Comment:
This will unfortunately not work, see https://github.com/aquaproj/aqua-renovate-config/issues/191#issuecomment-1211578160 . A workaround seems to be:

```suggestion
# renovate: datasource=go depName=golang.org/x/tools/cmd/goimports packageName=golang.org/x/tools
```